modules/network/mainnet.ts

```diff
@@ -183,10 +183,10 @@ export const mainnetNetworkConfig: NetworkConfig = {
     ],
     userStakedBalanceServices: [new UserSyncGaugeBalanceService()],
     /*
-    For sub-minute jobs we set the alarmEvaluationPeriod and alarmDatapointsToAlarm to 1 instead of the default 3. 
+    For sub-minute jobs we set the alarmEvaluationPeriod and alarmDatapointsToAlarm to 1 instead of the default 3.
```

Comment:
This empty spaces are run by vscode formatter using the prettier settings from `.prettierrc` 

Given that there are no lint checks in CI, I guess it is ok to leave them but tell me if you have a concrete local setup that I should use.

Reply:
Strange, I also use the `.prettierrc` in the rootfolder of this project and it doesnt remove the space. Whatever, that is fine.